cdap-docs/admin-manual/source/installation/azure-hdinsight.rst

```diff
@@ -0,0 +1,156 @@
+      :class: bordered-image
+
+      **Microsoft Azure:** Finding HDInsight
+
+#. Create an *HDInsight cluster* by clicking "Create". Enter a *Cluster name* (such as
+   *cdap-cluster*). Under *Cluster type*, select "HBase". (To continue, click *Select* to
+   close the *Cluster configuration* window):
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-2.png
+      :figwidth: 100%
+      :width: 800px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Creating a cluster
+
+#. Enter the required basic settings: *Cluster login password*, *Resource group*, and *Location*:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-3.png
+      :figwidth: 100%
+      :width: 674px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Configuring basic settings
+
+#. Under *Storage*, set a storage account, either an existing account or enter a unique name for a new account.
+
+#. Click *Next* to review the summary.  Then click *Create* to start cluster initialization:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-4.png
+      :figwidth: 100%
+      :width: 800px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Summary and confirming configuration
+
+#. Once the cluster is ready, its icon will show on the portal dashboard:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-5.png
+      :figwidth: 100%
+      :width: 176px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Cluster is created and running
+
+
+Install CDAP as an Application
+==============================
+#. Open the created HBase Cluster. Click the *Applications* button:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-6.png
+      :figwidth: 100%
+      :width: 800px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** The cluster dashboard
+
+#. In the open window, click *+ Add* to show the *Available applications*.  Choose the CDAP
+   version you'd like to install and accept the legal terms. Once confirmed in the following step,
+   installation of CDAP starts:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-7.png
+      :figwidth: 100%
+      :width: 800px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** Adding CDAP as an application
+
+#. The initial status starts at *Accepted*:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-8.png
+      :figwidth: 100%
+      :width: 631px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** CDAP accepted as an application
+
+#. Once CDAP is installed successfully,  its status shows as *Installed*:
+
+   .. figure:: ../_images/azure-hdinsight/azure-hdinsight-9.png
+      :figwidth: 100%
+      :width: 631px
+      :class: bordered-image
+
+      **Microsoft Azure HDInsight:** CDAP installed as an application
+
+
+Verification
+============
+#. Click the *Portal* button to launch the login window. Login by entering the cluster
```

Comment:
maybe the standard note about "it may take a few minutes while the YARN containers come up